Neos.Flow/Classes/Command/CoreCommandController.php

```diff
@@ -246,16 +246,16 @@ public function setFilePermissionsCommand($commandlineUser, $webserverUser, $web
      * flag this behavior can be changed, but changes will only be committed
      * if the working copy was clean before applying the migration.
      *
+     * @param string $package The key of the package to migrate
      * @param boolean $status Show the migration status, do not run migrations
      * @param string $packagesPath If set, use the given path as base when looking for packages
-     * @param string $packageKey If set, migrate only the given package
      * @param string $version If set, execute only the migration with the given version (e.g. "20150119114100")
      * @param boolean $verbose If set, notes and skipped migrations will be rendered
      * @param boolean $force By default packages that are not under version control or contain local changes are skipped. With this flag set changes are applied anyways (changes are not committed if there are local changes though)
      * @return void
      * @see neos.flow:doctrine:migrate
      */
-    public function migrateCommand($status = false, $packagesPath = null, $packageKey = null, $version = null, $verbose = false, $force = false)
```

Comment:
just moving making the package key param non-optional and renaming it to `package` for consistency with other CLI commands